### DISCUSSION TITLE
Option to save PHP variable in Latte tag to Javascript variable

### DISCUSSION THREAD

**Discussion by martinoak** [nette/latte#338] [Q&A]
Hi,
Currently I have `var cards = {$cards}` in .latte file, but in source code this shows huge array ($cards is a huge array). Can I somehow achieve to only pass `{$cards)` to `var cards` but **not** showing the variable in the source code of the website?

Thanks!

**Comment by MartinMystikJonas** [comment]
I am not sure what you want. PHP variable is stored in servers memory. If you want it to be used on website user's computer by JavaScript you have to pass it somehow. So it has to be in source code of website or has to be loaded by separate AJAX request. But it still has to be transfered to client in response body.
